Lib/test/_test_multiprocessing.py

```diff
@@ -4236,6 +4236,12 @@ def test_empty(self):
 
         proc.join()
 
+    def test_close(self):
+        queue = multiprocessing.SimpleQueue()
+        queue.close()
+        # closing a queue twice should not fail
+        queue.close()
```

Comment:
I dislike testing private attributes in unit tests. Should I add a second ``@cpython_only`` unit test testing private attributes?